hack/run-operating-system-manager.sh

```diff
@@ -27,5 +27,4 @@ $(dirname $0)/../_build/osm-controller \
   -namespace=cloud-init-settings \
   -worker-count=50 \
   -cni-version=v0.8.7 \
-  -containerd-version=1.4 \
   -cri-tools-version=v1.22.0
```

Comment:
hm...
looks like the same fate should be for -cni-version and -cri-tools-version flags

@moadqassem WDYT?

Reply:
Yeah I agree with Artiom. I mean why we introduced that as a parameter anyway 🤔 I might have overlooked it in the previous OSM ubuntu improvement

Reply:
@ahmedwaleedmalik  & @kron4eg I am working on a pr to adjust couple of things in the current OSPs. Will include in my PR thus we can ignore it for now.

Reply:
Okay thanks @moadqassem :+1: